.github/workflows/test.yml

```
@@ -28,46 +28,44 @@ jobs:
-            # On Windows, use PowerShell to get LOCALAPPDATA
-            PLAYWRIGHT_PATH=$(powershell -Command "Write-Output $env:LOCALAPPDATA\ms-playwright")
-            echo "path=$PLAYWRIGHT_PATH" >> $GITHUB_OUTPUT
+            echo "path=$HOME/.cache/ms-playwright" >> $GITHUB_OUTPUT
+          else
+            echo "path=$HOME/Library/Caches/ms-playwright" >> $GITHUB_OUTPUT
           fi
+        shell: bash
+
+      - name: Set Playwright cache path (Windows)
+        if: runner.os == 'Windows'
+        id: playwright-path
```

actionlint (1.7.10)

step ID "playwright-path" duplicates. previously defined at line:33,col:13. step ID must be unique within a job. note that step ID is case insensitive

(id)